src/apps/components/AppAlerts/useAppsFailedDeliveries.ts

```diff
@@ -12,36 +11,32 @@ interface AppsFailedDeliveries {
 }
 
 export const useAppsFailedDeliveries = (): AppsFailedDeliveries => {
-  const { hasManagedAppsPermission } = useHasManagedAppsPermission();
-
-  const [fetchAppsWebhooks, { data }] = useAppFailedPendingWebhooksLazyQuery();
+  const [fetchAppsWebhooks, { data }] = useAppFailedPendingWebhooksLazyQuery({
+    fetchPolicy: "no-cache",
+  });
 
   const lastFailedWebhookDate = useMemo(
     () =>
-      data?.apps?.edges.reduce(
-        (acc, app) => {
-          const latestFailedAttempt = getLatestFailedAttemptFromWebhooks(app.node.webhooks ?? []);
-
-          if (!latestFailedAttempt) {
-            return acc;
-          }
-
-          if (!acc) {
-            return latestFailedAttempt.createdAt;
-          }
-
-          return latestFailedAttempt.createdAt > acc ? latestFailedAttempt : acc;
-        },
-        null as Moment | null,
-      ) ?? null,
+      data?.apps?.edges.reduce((acc, app) => {
+        const latestFailedAttempt = getLatestFailedAttemptFromWebhooks(app.node.webhooks ?? []);
+
+        if (!latestFailedAttempt) {
+          return acc;
+        }
+
+        if (!acc) {
+          return latestFailedAttempt.createdAt;
+        }
+
+        return latestFailedAttempt.createdAt > acc ? latestFailedAttempt : acc;
+      }, null) ?? null,
     [data?.apps?.edges],
   );
 
   const handleFetchAppsWebhooks = () => {
-    // TODO: checking if webhooks should be fetched will be extracted out of this hook in a separate ticket
     fetchAppsWebhooks({
       variables: {
-        canFetchAppEvents: hasManagedAppsPermission,
+        canFetchAppEvents: true,
```

Comment:
question: Don't you need permission check?

Reply:
It's checked outside of this hook

Reply:
Maybe would be good to add short comment about this